pkg/operator/ceph/cluster/cluster.go

```diff
@@ -607,22 +607,31 @@ func (c *cluster) configureMsgr2() error {
 	}
 	monStore := config.GetMonStore(c.context, c.ClusterInfo)
 
```

Comment:
Instead of doing this can we move "rbd_default_map_options": defaultRBDMapOptions, from encryptionGlobalConfigSettings.  And just create it irrespective of whether encryption is enabled or disabled at the beginning in the function. As we might need such similar rbd map options for compression feature in the future, So might be good to keep the settings we need to configure msgr2 port separate from encryption setting, irrespective of encryption/compression enabled/disabled.

Reply:
>  As we might need such similar rbd map options for compression feature in the future

Also, compression is not supported by the kernel client.